providers/common/der/RSA.asn1

```diff
@@ -86,3 +86,14 @@ id-rsassa-pkcs1-v1_5-with-sha3-224 OBJECT IDENTIFIER ::= { sigAlgs 13 }
 id-rsassa-pkcs1-v1_5-with-sha3-256 OBJECT IDENTIFIER ::= { sigAlgs 14 }
 id-rsassa-pkcs1-v1_5-with-sha3-384 OBJECT IDENTIFIER ::= { sigAlgs 15 }
 id-rsassa-pkcs1-v1_5-with-sha3-512 OBJECT IDENTIFIER ::= { sigAlgs 16 }
+
+
+-- -------------------------------------------------------------------
+-- These OID's exist in the codebase but may need to be deprecated at some point.
+-- mdc2 and md5_sha1 have been omitted as they do not look like valid entries.
```

Comment:
Would you mind adding a reference to where you found this?

Reply:
Richard,

I used one of the oid repos for RIPE.
The closest thing I can find is https://homes.esat.kuleuven.be/~bosselae/ripemd160.html (which references iso standards which I cant see).
I can see some RFC https://www.ietf.org/rfc/rfc4051.txt that has references to dead links - so that is not really great either.
So the short answer is no :)
It is referred to in PKCS11 and CMS so it is probably being used by someone somewhere.

Reply:
Okie, I can deal with that.